app/src/main/java/com/imashnake/animite/features/MainActivity.kt

```diff
@@ -127,13 +136,24 @@ fun MainScreen(modifier: Modifier = Modifier) {
                 )
         )
 
-        AnimatedVisibility(
-            visible = isNavBarVisible,
-            modifier = Modifier.align(Alignment.BottomCenter),
-            enter = slideInVertically { it },
-            exit = slideOutVertically { it }
-        ) {
-            NavigationBar(navController = navController)
+        if (LocalConfiguration.current.orientation == Configuration.ORIENTATION_PORTRAIT) {
+            AnimatedVisibility(
+                visible = isNavBarVisible,
+                modifier = Modifier.align(Alignment.BottomCenter),
+                enter = slideInVertically { it },
+                exit = slideOutVertically { it }
+            ) {
+                NavigationBar(navController = navController)
```

Comment:
Can we just have the if/else inside AnimatedVisibility?

Reply:
The enter and exit animations are different, can't tell which one is better and why.